### PR TITLE
avm2: Avoid marking interface method definitions with VM_INTERNAL

### PR DESCRIPTION
**Pull request by Aaron1011**
Our asc.jar doesn't seem to apply a version suffix to namespaces for interface method definitions. This was causing these methods to get marked as VM_INTERNAL when we loaded playerglobals, preventing SWF from invoking these methods through the interface (e.g. having a variable of type `IEventDispatcher`, and calling `dispatchEvent` on it)

### CODE REVIEW

**Comment by Lord-McSweeney** [comment]
This fixes #14446 and fixes the regression in #12810

File: core/src/avm2/globals/flash/desktop/IFilePromise.as [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package flash.desktop {
+    import flash.utils.IDataInput;
+    import flash.events.ErrorEvent;
+
+    [API("668")] // AIR 2.0
+    public interface IFilePromise {
+
+        [API("668")]
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/desktop/IFilePromise.as:8]: Aren't these unnecessary since the interface itself is version-gated?

Comment (Aaron1011) [reply core/src/avm2/globals/flash/desktop/IFilePromise.as:8]: Probably, but it doesn't hurt to be sure

Comment (evilpie) [reply core/src/avm2/globals/flash/desktop/IFilePromise.as:8]: Personally I think we should avoid adding what is basically noise. If we have bugs with methods/properties being exposed unexpectedly those can be fixed.

**Review by Dinnerbone — APPROVED** [review]

**Pull request merged**